tests/test_secret_values.py

```diff
@@ -9,6 +9,7 @@
     SerializedSecret,
     get_all_secrets,
 )
+from tests.fake_secrets import FakeRequiredSecret
```

Comment:
Nice catch on this. I think the intent was for the assert on line 95 to be > 0 instead of > 1, since it is looking for `SomeRequiredSecret`. I think updating that value is preferable over an import that is otherwise unused.

Reply:
That makes a lot more sense. Updated